javascriptv3/example_code/ses/src/libs/sesUtils.js

```diff
@@ -0,0 +1,132 @@
+import {
```

Comment:
(Optional) It might be worth a comment explaining what this file is for. It took me a bit of poking around to realize it was a set of helper functions for your tests, and I imagine other users will wonder the same thing.